applications/HDF5Application/tests/test_HDF5Application.py

```diff
@@ -1,7 +1,9 @@
 import KratosMultiphysics
```

Comment:
do you intentionally not add the xdmf test to be executed here?

Also I think you could have a `test_HDF5Application_mpi` which populates the mpi-test-suites with the mpi tests

Reply:
it was a mistake, thanks for spotting it.